AUTHENTICATION.md

```diff
@@ -28,22 +22,25 @@ providing **Project ID** and **Service Account Credentials** directly in code.
 
 ### Google Cloud Platform environments
 
-While running on Google Cloud Platform environments such as Google Compute Engine, Google App Engine and
-Google Kubernetes Engine, no extra work is needed. The **Project ID** and **Credentials** and are 
-discovered automatically from the attached service account. Code should be written as if already authenticated.
+While running on Google Cloud Platform environments such as Google Compute Engine, Google App Engine
+and Google Kubernetes Engine, no extra work is needed. The **Project ID** and **Credentials** and
+are discovered automatically from the attached service account. Code should be written as if already
```

Comment:
Probably should get rid of Project ID here, as our GAPIC clients don't discover it, or at least they don't surface the projectID in any meaningful way.
```suggestion
and Google Kubernetes Engine, no extra work is needed. The **Credentials**
are discovered automatically from the attached service account. Code should be written as if already
```

Reply:
Ahhh I was hunting for these, good catch.